services/aws/s3.ts

```diff
@@ -1,29 +1,105 @@
 import { s3 } from "services/aws/index";
 import fs from "fs";
 import path from "path";
+import S3, { ManagedUpload } from "aws-sdk/clients/s3";
+import { AWSError } from "aws-sdk";
```

Comment:
I would import this from `serviecs/aws/index`.I think it would be cleaner if we import everything from `aws-sdk` in one file. Thoughts?

Reply:
Especially if there might be more AWS services that might use this

Reply:
good idea - done!